### PR TITLE
adding nix pills to external sources.

### PR DESCRIPTION
**Pull request by henrik-ch**
Addresses #639.

Please look at line [174](https://github.com/NixOS/nix.dev/blob/10b936f6d8b079cdfa75502481fe56453f301bfb/source/contributing/documentation.md?plain=1#L174) especially to make sure that it's correctly representing our intentions.

### CODE REVIEW

File: source/contributing/documentation.md [hunk 1]
```diff
@@ -168,6 +168,13 @@ Where to migrate what:
 [NixOS Wiki]: https://nixos.wiki/
 [NixCon 2015: Make Nix friendlier for Beginners]: https://media.ccc.de/v/nixcon2015-3-MakeNixfriendlierforBeginners#video
 
+### Nix Pills
+[Nix Pills](https://nixos.org/guides/nix-pills/) is a series of blog posts written to give an in-depth introduction and guide to Nix.
```
Comment (fricklerhandwerk) [inline source/contributing/documentation.md:172]:
```suggestion
[Nix Pills](https://nixos.org/guides/nix-pills/) is a series low-level tutorials on building software packages with Nix, showing in detail how Nixpkgs is made from first principles. 
```

File: source/contributing/documentation.md [hunk 2]
```diff
@@ -168,6 +168,13 @@ Where to migrate what:
 [NixOS Wiki]: https://nixos.wiki/
 [NixCon 2015: Make Nix friendlier for Beginners]: https://media.ccc.de/v/nixcon2015-3-MakeNixfriendlierforBeginners#video
 
+### Nix Pills
+[Nix Pills](https://nixos.org/guides/nix-pills/) is a series of blog posts written to give an in-depth introduction and guide to Nix. 
+It was originally written by Luca Bruno. 
+Work is currently being done to update the Nix Pills to be more up-to-date with the current state of Nix.
+Furthermore, work is underway to migrate the technical infrastructure of Nix Pills to improve maintainability and make it easier for others to contribute.
```
Comment (fricklerhandwerk) [inline source/contributing/documentation.md:175]: This needs instructions regarding what to do if you have some time.

- Make examples work correctly
- Add links to reference documentation where needed
- Update versions and names
- etc.

**Review by fricklerhandwerk — COMMENTED** [review]
Reviewed in Nix docs team meeting

File: source/contributing/documentation.md (hunk 2 above)
Comment (fricklerhandwerk) [reply source/contributing/documentation.md:175]: This should also link to the issue tracker, ideally with a query for `good first issue` labels.

File: source/contributing/documentation.md [hunk 3]
```diff
@@ -168,6 +168,18 @@ Where to migrate what:
 [NixOS Wiki]: https://nixos.wiki/
 [NixCon 2015: Make Nix friendlier for Beginners]: https://media.ccc.de/v/nixcon2015-3-MakeNixfriendlierforBeginners#video
 
+### Nix Pills
+[Nix Pills](https://nixos.org/guides/nix-pills/) is a series low-level tutorials on building software packages with Nix, showing in detail how Nixpkgs is made from first principles.
```
Comment (proofconstruction) [inline source/contributing/documentation.md:172]:
```suggestion
[Nix Pills](https://nixos.org/guides/nix-pills/) is a series of low-level tutorials on building software packages with Nix, showing in detail how Nixpkgs is made from first principles. 
```

**Pull request merged**